### DISCUSSION TITLE
Pull Shark stopped showing public details when I let it also show private ones

### DISCUSSION THREAD

**Discussion by ediosyncratic** [community/community#28583] [Other Feature Feedback, Questions, & Ideas]
In the pull shark badge, before I allowed display of private details, I could see two cfengine pull requests listed; but once (prompted to do so by a pop-up) I'd enabled display of private details, it no longer shows the details of those, they're just as "inaccessible" as the third that's been added.

In any case, although it would make sense to not give details to others of private repository pull requests, it could at least show *me* those details !  Arguably anyone with access to a private repository should be able to see the entries relating to that repo in my pull shark details.

**Comment by ediosyncratic** [comment]
If what's happening is that, now that I allow private repo details to be mentioned, the pull requests that qualified me for a badge are private ones, where previously a public pull request served that purpose, then perhaps the thing to do is:
* if I've qualified in public requests for a badge, show the public request that qualified me for it, at least to users not able to see the private request that would otherwise have qualified me for it;
* only show the "inaccessible" for badge-qualifying events if I've only qualified for it by counting private pull requests; and maybe even then show the first public pull request after that instead, to anyone who can't see the repo the private one was in.

**Reply by radar-blip** [reply]
Yes
